File: mylibrary/src/main/java/com/sk/mylibrary/utils/DirectSDK.java
package com.sk.mylibrary.utils;

import android.app.Activity;
import android.content.Context;


public class DirectSDK {
    private static DirectSDK mInstance;
    public static Context context;
    public String otp = "1234";
    public Activity activity;


    public static synchronized DirectSDK getInstance() {
        if (mInstance == null) {
            mInstance = new DirectSDK();
        }
        return mInstance;
    }

    public static void initialize(Context context1) {
        context = context1;
        mInstance = new DirectSDK();
    }

}

File: mylibrary/src/main/java/app/retailer/krina/shop/com/mp_shopkrina_retailer/utils/DirectSDK.kt
package app.retailer.krina.shop.com.mp_shopkrina_retailer.utils

import android.app.Activity
import android.content.Context

class DirectSDK {
    var otp: String = "1234"
    var activity: Activity? = null


    companion object {
        private var mInstance: DirectSDK? = null
        var context: Context? = null

        @get:Synchronized
        val instance: DirectSDK
            get() {
                if (mInstance == null) {
                    mInstance = DirectSDK()
                }
                return mInstance!!
            }

        fun initialize(context1: Context?) {
            context = context1
            mInstance = DirectSDK()
        }
    }
}